packages/lambda-powertools-logger/package.json

```diff
@@ -14,5 +14,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "jest": "^25.1.0"
```

Comment:
is this needed? I thought we just need to have jest installed as dev dependency at the root so the individual packages didn't need to have it

Reply:
accidental commit, sorry, removing